2018/13xxx/CVE-2018-13381.json

```diff
@@ -15,17 +15,11 @@
                     "product": {
```

Comment:
'https://fortiguard.com/advisory/FG-IR-18-387, https://fortiguard.com/advisory/FG-IR-20-232' does not match '^(ftp|http)s?://\\S+$'